privaterelay/context_processors.py

```diff
@@ -77,3 +50,43 @@ def _get_fxa(request):
         return fxa
     except AttributeError:
         return None
+
+
+@lru_cache(maxsize=None)
```

Comment:
If nothing else, I'd like to move the logic into a cached function like this so we can avoid the performance hit of all this logic on every request.

Reply:
OK, I did that in 50dbfab5.